src/gui/transferlistsortmodel.cpp

```diff
@@ -58,6 +58,18 @@ namespace
         return isLeftValid ? -1 : 1;
     }
 
+    int customCompare(const QStringList &left, const QStringList &right, const Utils::Compare::NaturalCompare<Qt::CaseInsensitive> &comparator)
```

Comment:
Recently, it seems to me a good idea to name variables of the functor type in the style of function names. For example, `compare` here. Then it would appear naturally at the calling place, e.g. `compare(left[i], right[i])`.

Reply:
Also it seems a good idea to place private members of functor type separately from other private fields and name them like private methods. E.g. `naturalCompare` instead of `m_naturalCompare`.

Reply:
>For example, compare here. Then it would appear naturally at the calling place, e.g. compare(left[i], right[i]).

OK, done.

>Also it seems a good idea to place private members of functor type separately from other private fields and name them like private methods. E.g. naturalCompare instead of m_naturalCompare.

I haven't got a strong preference on this.